src/main/java/br/com/excaladashboard/models/CategoriasExcala.java

```diff
@@ -0,0 +1,77 @@
+package br.com.excaladashboard.models;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "categorias_excala")
+public class CategoriasExcala {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "gestor_trafego_categoria", length = 150)
+    private String gestorTrafegoCategoria;
+
+    @Column(name = "vendedor_clt", length = 150)
+    private String vendedorClt;
+
+    @Column(name = "vendedor_pj", length = 150)
+    private String vendedorPj;
+
+    @Column(name = "indicador", length = 150)
+    private String indicador;
+
+    @Column(name = "franqueado", length = 150)
+    private String franqueado;
+
```

Comment:
adicionar uma coluna de lista de SubCategoria e usar as anotaçoes oneToMany e ManyToOne

```suggestion
private List<SubCategoria> subcategorias
```